src/constants/locConstants.ts

```diff
@@ -561,6 +561,9 @@ export let flavorDescriptionMssql = l10n.t(
 export let flavorDescriptionNone = l10n.t(
     "Disable intellisense and syntax error checking on current document",
 );
+export let autoDisableNonTSqlLanguageServicePrompt = l10n.t(
+    "Possible non-SQL Server SQL file detected. Automatically disable Intellisense for all non-SQL Server SQL files?",
```

Comment:
@croblesm, can you help refine this string to make sure the branding references are correct? Also, I feel like saying "SQL" so many times makes it a bit confusing.

Reply:
Looks like you already did in another comment; thanks!